### PR TITLE
Show all password reset links if there are multiple associated users

### PR DESCRIPTION
**Pull request by bethanyaconnor**
If there were multiple user associated with an email and an admin tried to generate a password reset link, no links would show up in the reset password tool.

Now:
Screenshot 2020-05-14 at 6 17 51 PM

## Links

- [jira](https://codedotorg.atlassian.net/browse/FND-1151)

## Testing story

manual testing and unit test

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/app/controllers/passwords_controller.rb [hunk 1]
```diff
@@ -45,6 +45,13 @@ def show_reset_url_if_admin
     if raw_token = resource.try(:raw_token)
       url = edit_password_url(resource, reset_password_token: raw_token)
       flash[:notice] = "Reset password link sent to user. You may also send this link directly: <a href='#{url}'>#{url}</a>".html_safe
+    elsif resource.child_users
+      notice = "Reset password link sent to user. You may also send the link directly:<br>"
+      resource.child_users.each do |user|
+        url = edit_password_url(user, reset_password_token: user.raw_token)
+        notice += "#{user.username}: <a href='#{url}'>#{url}</a><br>"
```
Comment (Hamms) [inline dashboard/app/controllers/passwords_controller.rb]: just for the sake of paranoia, can we explicitly html-escape the username here?

File: dashboard/app/models/user.rb [hunk 2]
```diff
@@ -1474,6 +1474,7 @@ def send_reset_password_for_users(email, users)
       raw, _enc = Devise.token_generator.generate(User, :reset_password_token)
       self.child_users = unique_users
       send_devise_notification(:reset_password_instructions, raw, {to: email})
+      return self
```
Comment (Hamms) [inline dashboard/app/models/user.rb:1477]: why is this necessary?

**Review by Hamms — APPROVED** [review]
couple nits, otherwise LGTM!

File: dashboard/app/models/user.rb (hunk 2 above)
Comment (bethanyaconnor) [reply dashboard/app/models/user.rb:1477]: We have to return the object that contains the `child_users` (see two lines up) in order to display the names. Previously it was returning the devise notification from the previous line.

Comment (Hamms) [reply dashboard/app/models/user.rb:1477]: there it is! I was trying to find that line but missed it. SGTM
